doc/source/cluster/kubernetes/doc_sanitize.cfg

```diff
@@ -18,10 +26,42 @@ replace: RAYJOB-SUBMIT-NAME
 regex: \d+m\d+s|\d+m|\d+s
 replace: TIME-DURATION
 
+[time-stamp]
+regex: \d{4}-\d{2}-\d{2}\s\d{2}:\d{2}:\d{2}[.,]?\d*|[A-Z][a-z]{2}\s[A-Z][a-z]{2}\s+\d{1,2}\s\d{2}:\d{2}:\d{2}\s\d{4}
+replace: TIME-STAMP
```

Comment:
Don't use `|`. Split it into multiple regex for readability. Like `[time-stamp]` and `[time-stamp-miliseconds]`